src/AddTodoForm.js

```diff
@@ -11,16 +11,12 @@ function AddTodoForm(props){
         event.target.reset();
     };
     return (
-       
             <form onSubmit={handleAddTodo}>
                 <label htmlFor ="TodoTitle">Title</label>
                 <input type="text" name="title" id='todoTitle'></input>
```

Comment:
todoTitle in the id attribute should be double quotes to maintain consistency. also can be self closing tag since nothing inside tags unlike label or button.